klaviyo/api_helper.py

```diff
@@ -200,7 +236,35 @@ def _public_request(self, path, querystring):
         url = '{}/{}?{}'.format(self.api_server, path, querystring)
         return self._request(self.HTTP_GET, url, request_type=self.PUBLIC)
 
-    def _request(self, method, url, params=None, data=None, request_type=PRIVATE):
+    def _public_post_request(self, method=HTTP_GET, params=None, path=None, is_test=False):
+        """Executes the request being made.
+
+        Args:
+            method (str): Type of HTTP request.
+            url (str): URL to make the request to.
+            params (dict or json): Body of the request.
```

Comment:
it looks like the callers of this function would always pass in a dict, json we'd expect to be a "str" that is a dict.